release/ray_release/util.py

```diff
@@ -121,7 +121,7 @@ def exponential_backoff_retry(
             retry_cnt += 1
             if retry_cnt > max_retries:
                 raise
-            logger.info(
+            logger.warn(
```

Comment:
do you know if `warn` will print to the logs? should I use `exception` or `error` instead cc @krfricke

Reply:
actually, gunna go for `exception`, if we have spammy logs due to this can revisit